src/os/bluestore/BlueStore.cc

```diff
@@ -17965,7 +18569,7 @@ int BlueStore::add_existing_bluefs_allocation(Allocator* allocator, read_alloc_s
     }
     for (auto itr = bluefs_extents.begin(); itr != bluefs_extents.end(); extent_count++, itr++) {
       //dout(5) << "BlueFS[" << extent_count << "] <" << itr.get_start() << "," << itr.get_len() << ">" << dendl;
-      allocator->init_rm_free(itr.get_start(), itr.get_len());
+      shared_alloc.a->init_rm_free(itr.get_start(), itr.get_len());
```

Comment:
Please revert the change as it breaks the function.
The function modifies a temp @allocator and should not operate on the global shared-allocator

Reply:
@benhanokh 
> The function modifies a temp @Allocator and should not operate on the global shared-allocator

Please fire a relevant ticket for this bug - commenting on a merged PR isn't a proper way of dealing with the issue.